src/config/server_config.rs

```diff
@@ -99,6 +101,10 @@ impl ServerConfig {
                 .unwrap_or_else(|_| "10000".to_string())
                 .parse()
                 .unwrap_or(10000),
+            rpc_timeout_ms: env::var("RPC_TIMEOUT_MS")
+                .unwrap_or_else(|_| "5000".to_string())
```

Comment:
Please document this option and it's default value so it is displayed in docs

https://docs.openzeppelin.com/relayer/0.1.0/